lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const types = require('../utils/types');
+
+function makeErrorMessageForGetWithDefault(property, isImportedGetWithDefault) {
+  return isImportedGetWithDefault
+    ? `Use \`this.${property}\` instead of \`getWithDefault(this, '${property}', ...)\``
+    : `Use \`this.${property}\` instead of \`this.getWithDefault('${property}', ...)\``;
+}
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Use the `||` operator instead of `getWithDefault` for more expected behaviors',
```

Comment:
Works for me in the README...also escaping the backlashes fails the build

```
/home/travis/build/ember-cli/eslint-plugin-ember/lib/rules/no-get-with-default.js
   7:7    error  Replace `"Use·\`||\`·or·the·ternary·operator·instead·of·\`getWithDefault()\`."` with `'Use·`||`·or·the·ternary·operator·instead·of·`getWithDefault()`.'`                                                                                                                                                                                                        prettier/prettier
```
https://travis-ci.org/ember-cli/eslint-plugin-ember/jobs/613841197?utm_medium=notification&utm_source=github_status

Reply:
The pipes are supposed to be escaped, not the backticks.

Try this:
```
const ERROR_MESSAGE = 'Use `\|\|` or the ternary operator instead of `getWithDefault()`';
```

And you can use `// eslint-disable-line no-useless-escape` if needed.